cmd/api.go

```diff
@@ -0,0 +1,27 @@
+package cmd
+
+import (
+	"log"
+	"net/http"
+	"github.com/spf13/cobra"
+	sw "github.com/kubefirst/kubefirst/internal/api"
+)
+
+var api = &cobra.Command{
+	Use:   "api",
+	Short: "starts API server",
+	Long:  "starts intermal API server",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		log.Printf("Server started")
+
+		router := sw.NewRouter()
+	
+		log.Fatal(http.ListenAndServe(":8081", router))
```

Comment:
panic is usually better than fatal, fatal will do a "force quit" `exit(1)` , and panic will call the `defers` / close open connection, files, etc before exiting.

Reply:
Good to know this. Great catch 🚀